etl/grapher_helpers.py

```diff
@@ -499,10 +499,15 @@ def _adapt_table_for_grapher(
 
 def _add_dataset_origins_to_variables(table: catalog.Table) -> catalog.Table:
     assert table.metadata.dataset
+    if not table.metadata.dataset.origins:
+        return table
+
     for col in table.columns:
         variable_meta = table[col].metadata
-        if not variable_meta.origins:
-            variable_meta.origins = table.metadata.dataset.origins
+        for origin in table.metadata.dataset.origins:
+            if origin not in variable_meta.origins:
+                variable_meta.origins.append(origin)
```

Comment:
Are you imposing that all origins in a dataset must be in all indicators? So, for example, if you have some origin about emissions and the population origin, you will have all indicators having both origins (even those that are not per capita). Or am I not understanding correctly?

Reply:
I'm going to fix this in subsequent PR. In short, `dataset.origins` shouldn't exist, but we should have an easy way how to specify origins for all indicators in YAML.